src/screens/AddScreen.js

```diff
@@ -67,6 +82,18 @@ export default function AddScreen({ navigation }) {
     if (checked) {
       result = { ...result, location: newLocation };
     }
+    result = {
```

Comment:
make this an "else" condition, so that this occurs only when geolocation is not checked. this is because if geolocation is checked we do not want latitude and longitude to be edited and it would be reduntant for this to occur